### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by rimbu002** [itmaybejj/editoria11y#23]
Hello,

We had our users test the module from an accessibility standpoint and they raised two issues:
1. Add a visible label to the Editoria11y button to make it more obvious what it represents.
2. The screenreader doesn’t read the label of the "?" tab if you arrow to it, but it does if you tab to it.

Thank you

**Comment by itmaybejj** [comment]
Good feedback; thank you. 

I'm going to merge these into #22 , since that will require refactoring the toolbar and and toggle icon.
